retrieveProducts/swagger-api.yaml

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
your swagger yaml contains a lot of errors, please fix that 
![image](https://user-images.githubusercontent.com/24233570/98799323-6400d500-2428-11eb-9959-8b9466c10d2c.png)

Reply:
This swagger yaml file was generated via AWS Api Gateway and is it valid in Postmat and in Api Gateway. I'm not sure why your tool show errors